tests/test_schema.py

```diff
@@ -1,11 +1,31 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
 import os.path
 
 from nomad.client import normalize_all, parse
 
 
+# @pytest.mark.skip(reason="These tests should only be for the schema validation and functionalities, not the actual data normalization and parsing")
```

Comment:
I agree there should be more tests for schema validation and functionalities as you said. This test can be a placeholder until we add more useful tests.

Reply:
Indeed 🙂